user_guide_src/source/libraries/curlrequest.rst

```diff
@@ -246,8 +246,10 @@ An example::
 debug
 =====
 
-When ``debug`` is passed and set to ``true``, this will enable additional debugging to echo to STDOUT during the
-script execution. This is done by passing CURLOPT_VERBOSE and echoing the output::
+When ``debug`` is passed and set to ``true``, this will enable additional debugging to echo to STDERR during the
+script execution. This is done by passing CURLOPT_VERBOSE and echoing the output. So, when you're running a build-in
```

Comment:
script execution. This is done by passing CURLOPT_VERBOSE and echoing the output. So, when you're running a **built-in**

Reply:
Thanks.